pkg/common/crypto/default_test.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import (
+	"testing"
+
+	"github.com/google/tink/go/aead"
+	"github.com/google/tink/go/keyset"
+	"github.com/google/tink/go/signature"
+	aeadsubtle "github.com/google/tink/go/subtle/aead"
+	"github.com/stretchr/testify/require"
+	chacha "golang.org/x/crypto/chacha20poly1305"
+
+	tinkcrypto "github.com/hyperledger/aries-framework-go/pkg/framework/provider/crypto"
+)
+
+func TestCrypto(t *testing.T) {
+	// Assert that MainCrypto implements the FrameworkCrypto interface.
+	var _ FrameworkCrypto = (*MainCrypto)(nil)
+}
```

Comment:
nit: use the global var and disable a linter `var _ FrameworkCrypto = (*MainCrypto)(nil)`

Reply:
removed test function